python/restate/context.py

```diff
@@ -57,13 +68,13 @@ class KeyValueStore(abc.ABC):
     @abc.abstractmethod
     def get(self,
             name: str,
-            serde: Serde[T] = JsonSerde()) -> Awaitable[Optional[Any]]:
+            serde: Serde[T] = JsonSerde()) -> RestateDurableFuture[Optional[Any]]:
```

Comment:
in other SDKs this is not combineable, because generally it's not something we expect people to do...